java/client/src/main/java/glide/api/commands/StringBaseCommands.java

```diff
@@ -74,6 +76,45 @@ public interface StringBaseCommands {
      */
     CompletableFuture<String> getdel(String key);
 
+    /**
+     * Gets the value associated with the given <code>key</code>. <code>GETEX</code>is similar to
+     * {@see #get}.
+     *
+     * @since Redis 6.0.6.
+     * @see <a href="https://redis.io/docs/latest/commands/getex/">redis.io</a> for details.
+     * @param key The <code>key</code> to retrieve from the database.
+     * @return If <code>key</code> exists, return the value of the <code>key</code>. Otherwise, return
+     *     <code>null</code>.
+     * @example
+     *     <pre>{@code
+     * String value = client.getex("key").get();
+     * assert value.equals("value");
+     *
+     * }</pre>
+     */
+    CompletableFuture<String> getex(@NonNull String key);
+
+    /**
+     * Gets the value associated with the given <code>key</code> and optionally sets its expiration.
+     * <code>GETEX</code> is similar to {@see #get}.
```

Comment:
should it be `@see` or `@link`

Reply:
i've done something like this before for my append command iirc